packages/scripts/scripts/start.js

```diff
@@ -30,9 +30,16 @@ process.env.WP_SRC_DIRECTORY = hasArgInCLI( '--webpack-src-dir' )
 	? getArgFromCLI( '--webpack-src-dir' )
 	: 'src';
 
+const webpackArgs = getWebpackArgs();
+if ( hasArgInCLI( '--hot' ) ) {
+	webpackArgs.push( 'serve' );
+} else if ( hasArgInCLI( '--no-watch' ) === false ) {
```

Comment:
It probably could be simplified to:
```suggestion
} else if ( ! hasArgInCLI( '--no-watch' ) ) {
```

Reply:
Thanks, done.